crowbar_framework/app/constraints/api_constraint.rb

```diff
@@ -15,13 +15,25 @@
 #
 
 class ApiConstraint
-  attr_reader :version
+  attr_reader :versions
 
-  def initialize(options)
-    @version = options.fetch(:version)
+  def initialize(*versions)
+    @versions = versions
   end
 
   def matches?(request)
-    request.headers.fetch(:accept) == "application/vnd.crowbar.v#{version}+json"
+    allowed = false
+    major, minor = nil
+    versions.each do |version|
+      major, minor = version.to_s.split(".").map(&:to_i)
+      version_mime = %r(^application/vnd\.crowbar\.v(?<major>\d+).(?<minor>\d+)\+json$)
+
+      versions_requested = version_mime.match(request.accept)
+      allowed = true if versions_requested &&
+          versions_requested[:major].to_i == major &&
+          versions_requested[:minor].to_i == minor
```

Comment:
Hey, having `==` here is exactly **not** the point!
I expect API v2.4 to be _superset_ of API v2.3, so when I ask for 2.3-compatible response, 2.4 will do!

(Also, don't forget to indicate the resulting version in the response headers.)